app/controllers/plans_controller.rb

```diff
@@ -235,9 +237,8 @@ def export
     @plan = Plan.find(params[:id])
     authorize @plan
 
-    # If no format is specified, default to PDF
-    params[:format] = 'pdf' if params[:format].nil?
-
+    # We should re-work this into something more useful than creating a new one
+    # every time a plan gets exported
```

Comment:
Yes I think this all ties in with the statistics discussion. Tagging issues #154 and #346